.github/workflows/cut-release.yml

```diff
@@ -0,0 +1,60 @@
+name: Cut Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        required: true
+        type: string
+        description: 'Release tag'
+      key_ring:
+        required: true
+        type: string
+        description: 'Key ring for cosign key'
+      key_name:
+        required: true
+        type: string
+        description: 'Key name for cosign key'
+
+concurrency: cut-release
+
+jobs:
+  cut-release:
+    name: Cut release
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: write
+    env:
+      GIT_TAG: ${{ github.event.inputs.release_tag }}
+      PROJECT_ID: 'projectsigstore'
+    steps:
+      - name: Check actor access
+        if: ${{ !contains(fromJson('["bobcallaway","cpanato","dlorenc","lukehinds"]'), github.actor) }}
+        run: exit 1
+
+      - name: Checkout out repo
+        uses: actions/checkout@v2
+        with:
+          path: ./src/github.com/sigstore/fulcio
+
+      - name: Draft release
+        working-directory: ./src/github.com/sigstore/fulcio
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: gh release create --repo github.com/sigstore/fulcio --draft ${{ env.GIT_TAG }}
```

Comment:
we don't need this, the goreleaser that will run in the cloudbuild job will create the gh release for us

Reply:
then maybe we can remove the `contents` permission

Reply:
thanks for pointing that out.
removed.